helix-term/src/ui/document.rs

```diff
@@ -287,9 +287,11 @@ pub fn render_text<'t>(
             style_span.0
         };
 
+        let virt = grapheme.is_virtual().clone();
```

Comment:
```suggestion
        let virt = grapheme.is_virtual();
```

Doesn't really make a difference but cloning a `bool` is kind of pointless

Reply:
Had this as a relic when I was working with &bool